test/test_light.jl

```diff
@@ -1,22 +1,24 @@
 using Oceananigans, Test
-using OceanBioME: TwoBandPhotosyntheticallyActiveRadiation, LOBSTER
-using Oceananigans.Biogeochemistry: update_biogeochemical_state!
+using OceanBioME: TwoBandPhotosyntheticallyActiveRadiation, LOBSTER, NutrientPhytoplanktonZooplanktonDetritus
+using Oceananigans.Biogeochemistry: update_biogeochemical_state!, required_biogeochemical_tracers
 
-grid = RectilinearGrid(size=(1,1,2), extent=(1,1,2))
+Pᵢ(x,y,z) = 2.5 + z
 
-@testset "Two band attenuation" begin
+function test_two_band(grid, bgc, model_type)
+    biogeochemistry = bgc(; grid,
+                            light_attenuation_model = TwoBandPhotosyntheticallyActiveRadiation(; grid),
+                            surface_phytosynthetically_active_radiation = (x, y, t) -> 100.0)
 
-    model = NonhydrostaticModel(; grid, 
-                                  biogeochemistry = LOBSTER(; grid,
-                                                              light_attenuation_model = TwoBandPhotosyntheticallyActiveRadiation(; grid),
-                                                              surface_phytosynthetically_active_radiation = (x, y, t) -> 100.0))
-    Pᵢ(x,y,z) = 2.5 + z
+    model = model_type(; grid, 
+                         biogeochemistry,
+                         tracers = unique((required_biogeochemical_tracers(biogeochemistry)..., :T, :S))) # because hydrostatic free surface will request T and S and some BGC models will too
```

Comment:
```suggestion
                         tracers = unique((required_biogeochemical_tracers(biogeochemistry)..., :T, :S))) # because hydrostatic free surface model will request T and S and some BGC models will too
```